src/commands/game/songcheck.js

```diff
@@ -1,59 +1,61 @@
+'use strict';
+
 // Dependencies
 const config = require('../../../config');
 const Discord = require('discord.js');
 
-module.exports = {
```

Comment:
Avoid putting everything inside `module.exports` as it reads harder.

`module.exports` should only define what is exported; no need to include the content in there.

This pretty much applies to all command files, as seen below.

Reply:
Definitely improves clarity